docs/services/teams-workflows.md

```diff
@@ -0,0 +1,228 @@
+# Teams Workflows
+
+## Overview
+
+The Teams Workflows notification service sends message notifications using Microsoft Teams Workflows (Power Automate). This is the recommended replacement for the legacy Office 365 Connectors service, which will be retired on March 31, 2026.
+
+## Parameters
+
+The Teams Workflows notification service requires specifying the following settings:
+
+* `recipientUrls` - the webhook url map, e.g. `channelName: https://api.powerautomate.com/webhook/...`
+
+## Supported Webhook URL Formats
+
+The service supports the following Microsoft Teams Workflows webhook URL patterns:
+
+- `https://api.powerautomate.com/...`
+- `https://api.powerplatform.com/...`
+- `https://flow.microsoft.com/...`
+- `https://webhook.office.com/workflows/...`
+- URLs containing `/powerautomate/` in the path
+
+## Configuration
+
+1. Open Microsoft Teams and navigate to the channel where you want to receive notifications
+2. Click **Workflows** in the channel menu
+3. Click **Create** and select **When a webhook request is received**
```

Comment:
@pasha-codefresh I just looked into it configuring the same. I couldn't see to select **When a webhook request is received**. Can you help here?

<img width="1193" height="769" alt="Image" src="https://github.com/user-attachments/assets/3cfb3ac7-b966-479f-81d6-d248ddd975a4" />